include/powsybl/iidm/Extendable.hpp

```diff
@@ -53,6 +54,16 @@ class Extendable {
 
     stdcxx::range<Extension> getExtensions();
 
```

Comment:
I chose not to add `getImplementationName()` here, because it is not relevant in C++ context. 
Do you think I should (as non virtual) ?

Reply:
No I think it's a good idea to remove this method